tests/php/modules/shortcodes/test_class.soundcloud.php

```diff
@@ -24,4 +24,36 @@ public function test_shortcodes_soundcloud() {
 		$this->assertNotEquals( $content, $shortcode_content );
 	}
 
+	public function test_shortcodes_soundcloud_html() {
+		$content = '[soundcloud url="https://api.soundcloud.com/tracks/156661852" params="auto_play=false&amp;hide_related=false&amp;visual=true" width="100%" height="450" iframe="true" /]';
+
+		$shortcode_content = do_shortcode( $content );
+
+		$this->assertContains( '<iframe width="100%" height="450"', $shortcode_content );
```

Comment:
That is ok. I tested it with `assertStringStartsWith` and it works in the same way.